src/components/EditSections/EditUserInfo/EditUserInfo.js

```diff
@@ -316,6 +340,20 @@ class EditUserInfo extends React.Component {
                 fullWidth
               />
             </Col>
+            <Col xs={12} md={3}>
+              <Field
+                label={<FormattedMessage id="ui-users.information.userType" />}
+                name="type"
+                id="userType"
+                component={Select}
+                fullWidth
+                initialValue={initialValues.type}
```

Comment:
Actually, in the previous comment, I meant to define the `type` value in the place where `initialValues` are defined for the form, but not in the field itself :)

Reply:
Think it would be enough https://github.com/folio-org/ui-users/pull/2546/files#diff-9e67afd51e4896f499c735e8c0be2e50fc7aecb012aa8c8e66b847713c29febaR87